File: src/main/java/ru/citeck/idea/references/resolvers/XmlAttributeValueValueResolver.java
package ru.citeck.idea.references.resolvers;

import com.intellij.psi.xml.XmlAttributeValue;
import ru.citeck.idea.references.PsiElementValueResolver;

public class XmlAttributeValueValueResolver implements PsiElementValueResolver<XmlAttributeValue> {

    @Override
    public Class<XmlAttributeValue> getPsiElementType() {
        return XmlAttributeValue.class;
    }

    @Override
    public Object getValue(XmlAttributeValue psiElement) {
        return psiElement.getValue();
    }
}

File: src/main/kotlin/ru/citeck/idea/references/resolvers/XmlAttributeValueValueResolver.kt
package ru.citeck.idea.references.resolvers

import com.intellij.psi.xml.XmlAttributeValue
import ru.citeck.idea.references.PsiElementValueResolver

class XmlAttributeValueValueResolver : PsiElementValueResolver<XmlAttributeValue> {

    override fun getPsiElementType(): Class<XmlAttributeValue> {
        return XmlAttributeValue::class.java
    }

    override fun getValue(psiElement: XmlAttributeValue): Any {
        return psiElement.value
    }
}
